entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntitiesEventService.kt

```diff
@@ -0,0 +1,28 @@
+package com.egm.stellio.entity.service
+
+import com.egm.stellio.shared.model.EntitiesEvent
+import com.egm.stellio.shared.util.JsonLdUtils.mapper
```

Comment:
not so nice to expose publicly such an internal object (I remember we have already talked about this)

Reply:
Maybe it can be protected since the JsonUtilsTests should use the same mapper